.github/workflows/pr.yml

```diff
@@ -135,6 +135,16 @@ jobs:
       - name: Bootstrap test
         run: SSL=3.0.x .ci-scripts/test-bootstrap.sh
 
+  fedora42-bootstrap:
+    name: Fedora 42 bootstrap
+    runs-on: ubuntu-latest
+    container:
+        image: ghcr.io/ponylang/ponyup-ci-fedora42-bootstrap-tester:latest
```

Comment:
We will need to change this to a specific date if/when this is accepted and an image is built and pushed.